### DISCUSSION TITLE
Add reference to the template in the developer's guide

### DISCUSSION THREAD

**Discussion by ninad-kamat** [ansys/pyansys-dev-guide#11] [General]
@akaszynski 
Since, the new template repository is ready now, can we add a reference to that in the developer's guide, especially in [Library Overview](https://github.com/pyansys/about/tree/main/doc/source/library_description) section?

**Comment by akaszynski** [comment]
Agreed, that should be there.

**Reply by akaszynski** [reply]
Added in #12
